app/src/main/java/com/example/campusguide/BottomNavigation.kt

```diff
@@ -40,4 +41,13 @@ class BottomNavigation constructor (
     private fun startNavigation() {
         directions.startFlow()
     }
+
+    private fun choosePOI() {
+        activity.permissions = Permissions(activity)
+
+        val locationProvider = FusedLocationProvider(activity)
```

Comment:
Import missing for FusedLocationProvider

Reply:
Yeah i fixed that happened when I pulled form master. I added it